### PR TITLE
[LMBiSeNet] Implement bilinear upsampling post process

### PR DESCRIPTION
**Pull request by ruimashita**
## Motivation and Context

For new semantic segmentation model `LMBiSeNet` which has bilinear sampling before softmax.
Because it is easy, I implement `Bilinear` class in `lmnet/post_processor.py`, not in C++ runtime.

## Description

I define the `compatible_tensorflow_v1` flag parameter in `Bilinear` class, When the flag is True, it is compatible with tensorflow v1 resize function. Otherwise tensorflow v2 and Pillow resize function.
Tensorflow v1 image resize function `tf.image.resize_bilinear()` which is not proper of calculataion of center pixel. The bug is fixed in tensorflow v2 `tf.image.resize()` which given the same result as Pillow's resize.
See also
* https://github.com/tensorflow/tensorflow/issues/6720
* https://github.com/tensorflow/tensorflow/commit/3ae2c6691b7c6e0986d97b150c9283e5cc52c15f

I implement 2 unit tests 
* Verify Bilinear post process results are same as tensorflow v1 tf.image.resize_bilinear()
* Verify Bilinear post process results are same as Pillow resize()

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: lmnet/lmnet/post_processor.py [hunk 1]
```diff
@@ -311,3 +313,106 @@ def __call__(self, outputs, **kwargs):
...
+        for h in range(output_height):
+            for w in range(output_width):
+                if compatible_tensorflow_v1:
+                    center_y = h * scale[0]
+                    center_x = w * scale[1]
+                else:
+                    center_y = (h + 0.5) * (scale[0]) - 0.5
+                    center_x = (w + 0.5) * (scale[1]) - 0.5
+
+                if compatible_tensorflow_v1:
```
Comment (lm-jira) [inline lmnet/lmnet/post_processor.py]: I think this `if` can be merged with `if` at line 386.

**Comment by lm-jira** [comment]
I added `Bilinear` postprocessor in object detection model.
It works without problem.

File: lmnet/lmnet/post_processor.py (hunk 1 above)
Comment (ruimashita) [reply lmnet/lmnet/post_processor.py]: sure

Comment (ruimashita) [reply lmnet/lmnet/post_processor.py]: Fixed in https://github.com/blue-oil/blueoil/pull/211/commits/3ad532032db1b916c93865f4384180b15f672601

**Comment by ruimashita** [comment]
@lm-jira @iizukak 
I deleted debug comment https://github.com/blue-oil/blueoil/pull/211/commits/3b44c63e92c63b225f6245fa3562f482b2cb016c, and  remove redundant condition https://github.com/blue-oil/blueoil/pull/211/commits/3ad532032db1b916c93865f4384180b15f672601.

Please review again 👍
